server/models/requestModel.js

```diff
@@ -0,0 +1,37 @@
+const mongoose = require("mongoose");
+
+const requestSchema = new mongoose.Schema({
+  userId: {
+    type: Schema.Types.ObjectId,
+    ref: "Profile",
+    required: true,
+  },
+  sitterId: {
+    type: String,
```

Comment:
This should also be a ref to profile

Reply:
Ler's also add indices to both sitterid and userId. Maybe rename the latter to ownerId to make it more clear